spec/helpers/application_helper/buttons/storage_scan_spec.rb

```diff
@@ -6,25 +6,19 @@
   let(:button) { described_class.new(view_context, {}, {'record' => record}, props) }
   let(:ems) { nil }
 
-  it_behaves_like 'a generic feature button after initialization'
+  it_behaves_like 'a feature button with disabled'
 
-  context 'when feature is not supported' do
-    it_behaves_like 'GenericFeatureButtonWithDisabled#calculate_properties'
-  end
-
-  context 'when feature is supported' do
-    describe '#disabled?' do
-      context 'but no EMSs are present' do
-        it_behaves_like 'a disabled button', 'Smartstate Analysis cannot be performed on selected Datastore'
-      end
-      context 'but no EMS has valid credentials for the Datastore' do
-        let(:ems) { FactoryBot.create(:ems_vmware) }
-        it_behaves_like 'a disabled button', 'There are no EMSs with valid credentials for this Datastore'
-      end
-      context 'with valid credentials for this Datastore' do
-        let(:ems) { FactoryBot.create(:ems_vmware_with_authentication) }
-        it_behaves_like 'an enabled button'
-      end
+  describe '#disabled?' do
+    context 'when no EMSs are present' do
+      it_behaves_like 'a disabled button', 'Smartstate Analysis cannot be performed on selected Datastore'
+    end
+    context 'when no EMS has valid credentials for the Datastore' do
```

Comment:
Minor rubocop, but there should be a blank line between contexts (also a few lines down)